### ISSUE TITLE
Remove debug logs

### ISSUE THREAD

**Issue by natalia-osa** [TimOliver/TOSMBClient#5]
Each ~5s the console prints:
`netbios_ns_send_name_query, name query sent for '*'.`
The logs are buried inside C files if the `libdsm` library as `BDSM_dbg`. After a minute, nothing can be seen in the console. Is there any way to turn them off? 

Btw is there any way to change the period between each discovery request sent?

**Comment by TimOliver** [comment]
Hi there Natalia!

Hahah, yeah I've noticed that myself, but it never really bothered me to the point where I wanted to fix it. 
I just looked into it, and it's a simple build setting macro. I've rebuilt libdsm with that macro and committed it here, so the next time you update it, the console logs will go away.

Yeah, it's possible to explicitly set the timeout between NetBIOS discovery events. It's one of the method arguments when you go to start it:

``` objc
- (BOOL)startDiscoveryWithTimeOut:(NSTimeInterval)timeout
                            added:(TONetBIOSNameServiceDiscoveryEvent)addedHandler
                          removed:(TONetBIOSNameServiceDiscoveryEvent)removedHandler;
```

Hope that helped!
